src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(...args) {
```

Comment:
just use one value instead of args
```suggestion
  return function adder(number) {
```